code/game/machinery/cryopod.dm

```diff
@@ -305,29 +249,23 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 
 	visible_message("<span class='notice'>[src] hums and hisses as it moves [mob_occupant.real_name] into storage.</span>")
 
-	for(var/obj/item/item in mob_occupant.GetAllContents())
-		if(item.loc.loc && (item.loc.loc == loc || item.loc.loc == control_computer))
-			continue // means we already moved whatever this thing was in
-			// I'm a professional, okay
-
-		if(!should_preserve_item(item))
+#if MIN_COMPILER_VERSION >= 514
+	#warn Please replace the loop below this warning with an `as anything` loop.
+#endif
+	for(var/mob_content in mob_occupant)
+		var/obj/item/item_content = mob_content
+		if(!istype(item_content))
 			continue
```

Comment:
How does this interact with storage implants and the like?

Reply:
im curious to see if it spills out all ur organs